Makefile

```diff
@@ -30,6 +33,10 @@ help:
 addsite:
 	@docker-compose exec zotonic bin/zotonic addsite -s ginger -H $(name).docker.test $(name)
 
+api-docs:
+	@yaml2json docs/rest-api.yaml > tmp/rest-api.json
```

Comment:
Why not simply a local `tmp` dir, such as `/tmp`? It seems unneeded to check in `tmp/` into the repository.